.github/workflows/podman-e2e-windows.yaml

```diff
@@ -65,6 +65,8 @@ jobs:
         podman logs x-session
         
     - name: Run podman desktop e2e
+      env:
```

Comment:
I do no think this env. var will be of use here. These are not "our" tests. these podman e2e functional tests.